src/collection/dimensions.js

```diff
@@ -925,6 +941,10 @@ defineDimFns( {
   name: 'height'
 } );
 
+defineDimFns( {
```

Comment:
The padding function should be defined manually.  `defineDimFns()` is only for brevity in defining width and height.  It may seem to work, but there are cases where using this for padding with break things.